source/1_engine/rex_engine/include/rex_engine/diagnostics/stacktrace.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+#ifdef REX_PLATFORM_WINDOWS
+  #include "rex_engine/diagnostics/win/win_stacktrace.h"
```

Comment:
I think this file belongs in some platform layer no?
Since it's windows only, if this makes it to confusing you can leave it here just something to consider.
I posted a lot more comments like this within the review the same applies to them.